jormungandr-lib/src/interfaces/block0_configuration/DOCUMENTED_EXAMPLE.yaml

```diff
@@ -84,7 +84,7 @@ blockchain_configuration:
 
   # initial value the treasury will start with, if not set the treasury
   # starts at 0
-  treasury: 1000000000000
+  treasury: 0
```

Comment:
I don't understand why you needed to change this value?

Reply:
The treasury initially starts empty.  There is no reason for it to contain anything.  Value is transferred to it from the reserves through tax

Reply:
this is a `DOCUMENTED_EXAMPLE.md` not the settings for the testnet.